src/services/WorkspaceFlowRunsApi.ts

```diff
@@ -72,16 +70,40 @@ export class WorkspaceFlowRunsApi extends WorkspaceApi implements IWorkspaceFlow
     return data
   }
 
-  public async getFlowRunsGraph(flowRunId: string): Promise<GraphNode[]> {
-    const { data } = await this.get<FlowRunGraphResponse[]>(`/${flowRunId}/graph`)
-
-    return mapper.map('FlowRunGraphResponse', data, 'GraphNode')
-  }
-
-  public async getFlowRunsTimeline(id: string): Promise<GraphTimelineNode[]> {
-    const { data } = await this.get<FlowRunGraphResponse[]>(`/${id}/graph`)
+  public async getFlowRunsGraph(id: string): Promise<RunGraphData> {
+    // todo: should we just send this through the mapper?
+    // context: the idea is that this would be more efficient since the response can be very large.
+    // But that is unproven from a perf perspective. However this is much simpler than a dedicated mapper.
+    // But generally consistency is better so should test if this even makes a difference perf wise.
```

Comment:
@pleek91 idk if we want to resolve this comment or not

Reply:
Its more a plead for others opinions lol. Unless someone thinks the mapper would be better I'd fine with shipping this.

Reply:
It should be faster because it maps while converting from json rather than going through the whole response twice (once to convert from json and once to map)